activity/settings.gradle

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+plugins {
+    id "com.gradle.enterprise" version "3.7.1"
+}
```

Comment:
It would be great to add
```
id 'com.gradle.common-custom-user-data-gradle-plugin" version "1.4.2"
```
as well, since this will enhance the builds scans with a bunch of tags and custom values. For example:
- link to all build scans with the same commit id
- tag for CI vs LOCAL (to make these easy to filter)
- link to all build scans from the same workflow or run